presto-product-tests/conf/docker/singlenode-hdfs-impersonation/docker-compose.yml

```diff
@@ -1,35 +1,7 @@
 version: '2'
 services:
-  hadoop-master:
-    extends:
-      file: ../common/standard.yml
-      service: hadoop-master-common
 
   presto-master:
-    extends:
-      file: ../common/standard.yml
-      service: presto-master-common
     command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode run
```

Comment:
I would move it to `standard.yaml`. It is different in kerberos/multinode and it is already overwritten there.